webpack.config.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
     alias: {
       components: path.resolve(__dirname, 'src/components'),
       state: path.resolve(__dirname, 'src/state'),
-      styles: path.resolve(__dirname, 'src/components/styles'),
+      styles: path.resolve(__dirname, 'src/assets/styles'),
```

Comment:
I realize we don't have a `components/styles` folder anymore but I think naming this folder styles is confusing... Technically assets should never hold any styles... Can we rename this to something more specific like `variables` / `global-styles`

Reply:
I'd still rather keep them under `assets` to avoid more root nodes. Also would like to treat `assets` more than just a dir for images. Common styles and colors can be treated as an asset.

Reply:
Oh we should def keep it under assets but just rename it to something other than styles

Reply:
If its `global-styles` in assets I feel it is redundant. A dir called `styles` in assets is by default a global file to be used across the project.

Reply:
yeah good point, lets name it something else then

Reply:
`styles` make sense, why would you name it something else?

Reply:
because it's not holding styles! What makes it more confusing is that you also call the alias `styles` which you use within our actual styles!